### ISSUE TITLE
Comments before stream starts

### ISSUE THREAD

**Issue by remlap** [xenova/chat-replay-downloader#26]
I was puzzled as to why comments were not grabbing for ones written before stream starts, it then dawned on I needed to set a minus number start time to grab them all.

Could this be added to the help.

Cheers

**Comment by xenova** [comment]
Sure I can do this 👍

**Comment by xenova** [comment]
Actually, it might be best to, by default, get all messages (even those before starting). And then, the user can specify a start time of 0 if they want to get only the messages after the stream has started. Thoughts?

**Comment by remlap** [comment]
Much better way imo.

**Comment by xenova** [comment]
Just an update on this: I have been working on it in a different branch, along with a lot of other changes to the code: https://github.com/xenova/chat-replay-downloader/tree/package

However, this branch is not ready yet to merge into the master branch yet. But this change will be made in that update! 😃

**Comment by xenova** [comment]
Alright - finally merged the `package` branch into `master`. This should close the issue.
